src/widget/wtrackmenu.cpp

```diff
@@ -573,14 +563,19 @@ void WTrackMenu::updateMenus() {
                 }
             }
         }
-        m_pBpmUnlockAction->setEnabled(anyBpmLocked);
-        m_pBpmLockAction->setEnabled(!anyBpmLocked);
-        m_pBpmDoubleAction->setEnabled(!anyBpmLocked);
-        m_pBpmHalveAction->setEnabled(!anyBpmLocked);
-        m_pBpmTwoThirdsAction->setEnabled(!anyBpmLocked);
-        m_pBpmThreeFourthsAction->setEnabled(!anyBpmLocked);
-        m_pBpmFourThirdsAction->setEnabled(!anyBpmLocked);
-        m_pBpmThreeHalvesAction->setEnabled(!anyBpmLocked);
+        if (featureIsEnabled(Feature::Reset)) {
+            m_pClearBeatsAction->setEnabled(!anyBpmLocked);
```

Comment:
Somewhat off topic, but I think it would be convenient to duplicate the reset BPM action in the Adjust BPM menu. I stumbled upon this myself just now.

Reply:
Could you please add that in this PR?

Reply:
Added.